dc/s2n-quic-dc/events/connection.rs

```diff
@@ -0,0 +1,14 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#[event("application:write")]
+pub struct ApplicationWrite {
+    /// The number of bytes that the application tried to write
+    len: usize,
+}
+
+#[event("application:write")]
+pub struct ApplicationRead {
+    /// The number of bytes that the application tried to read
+    len: usize,
```

Comment:
nit: maybe capacity is a better name for this field, since I presume it's how much space the application provided us?

Reply:
yeah that's a good call. i can fix when i wire it up